### DISCUSSION TITLE
How to deploy a contract to the in-memory local node via hardhat

### DISCUSSION THREAD

**Discussion by Gilgamesh-818** [zkSync-Community-Hub/zksync-developers#67] [General] [answered]
What should my hardhat.config.ts file look like? How do I specify an ethNetwork rpc url when I'm only trying to deploy to the layer 2? Is there any way to do this?

**Comment by MexicanAce** [comment]
To run against the in-memory node, your network in `hardhat.config.ts` should look like the following:

```
{
    url: "http://localhost:8011",
    ethNetwork: "", #You can set this URL, but it's not necessary
    zksync: true,
}
```
 will just need to update the `url` to `http://localhost:8081` and set the

**Reply by uF4No** [reply, marked as answer]
Just adding that [the network details are available in our docs](https://era.zksync.io/docs/tools/testing/era-test-node.html#network-details)
